UofS-Pulse-Binfo/analyzedphenotypes
# ncit__method (methods) and sio__study (experiments) fields

## Metadata

 - Issue #61 




Documentation:

## Description
This PR creates two new fields for use on Trait pages:
 - ncit__method: lists all available methods for a given trait including units.
![screen shot 2019-02-18 at 5 03 27 pm](https://user-images.githubusercontent.com/1566301/52980340-385dd180-339f-11e9-9594-3463ad08160e.png)
![screen shot 2019-02-18 at 5 03 34 pm](https://user-images.githubusercontent.com/1566301/52980342-398efe80-339f-11e9-82dc-ef69884f8dd4.png)
 - sio__study: lists all experiments with data for a given trait.
![screen shot 2019-02-18 at 5 03 40 pm](https://user-images.githubusercontent.com/1566301/52980345-401d7600-339f-11e9-914f-6ed3b70a7353.png)
![screen shot 2019-02-18 at 5 03 48 pm](https://user-images.githubusercontent.com/1566301/52980349-43186680-339f-11e9-951b-a4d4ee59e5f1.png)
Trait/2653817
## Testing?




1. Navigate to structure > Tripal Content Types > [cvterm-based content type] > manage fields
2. Check for new fields and confirm ncit__method and sio__study were added
3. Go to manage display and ensure both fields are displayed
4. Go to a specific trait page and ensure you see all methods and experiments.

NOTE: On KP Fresh the correct content type is 'Trait' and it stores cvterm in 'UofS Lentil Traits' for project genus 'Lens'. Example trait pages are: Trait/2653817, Trait/2653826, Trait/2653829
NOTE2: in the examples provided some traits have unintelligible names. This is due to them being loaded before PR #60 was implemented.
